.github/workflows/ci.yml

```diff
@@ -65,7 +65,7 @@ jobs:
 
       - name: '@grafana/sign-plugin - sign generated plugin'
         env:
-          GRAFANA_API_KEY: ${{ secrets.GRAFANA_API_KEY }}
+          GRAFANA_ACCESS_POLICY_TOKEN: ${{ secrets.GRAFANA_ACCESS_POLICY_TOKEN }}
```

Comment:
To make sure the GRAFANA_API_KEY still work. can you add a new step that uses `GRAFANA_API_KEY` ? just underneath this one. you can adjust the `name` so it is clear one works with policy token and a nother with api_key

Reply:
I think there is still the previous step `use GRAFANA_API_KEY to sign generated plugin` below this one?